### DISCUSSION TITLE
Acaba l'sprint 1 i comença l'sprint 2.

### DISCUSSION THREAD

**Discussion by FherStk** [ElPuig/IMS#4] [1. IMS: general]
Bon dia a tothom,

Perdoneu el retard, però l'sprint 1 (temps per a preparar l'entorn de desenvolupament i fer proves amb Odoo) va acabar el dia 28 i el dia 1 va començar l'sprint 2. Com ja sabeu, hem definit els sprints perquè tinguin una durada d'un mes i durant aquest segon sprint ens toca el següent:

1. Detallar les funcionalitats dels dos components que volem desenvolupar amb prioritat, detallant què farà i com ho farà: 
1.1 Control d'assistència i fitxa d'alumne: https://github.com/ElPuig/IMS/discussions/2 
1.2 Gestió d'actes: https://github.com/ElPuig/IMS/discussions/3
3. Detallar els models de dades que necessitem: https://github.com/ElPuig/IMS/discussions/1

Aquests temes haurien de quedar tancats per poder començar l'sprint 3 el dia 1 d'abril. 

Salut i gràcies a tots,
Fer

**Comment by jherna** [comment]
Proves fetes!

He provat de fer mòduls i fer connexions en remot des del VSCode com diu 
la wiki i també amb PyCharm. Tot ok!!! La única cosa que ja vaig 
comentar que em generava error era el path_module que havia de posar-ho 
amb barra baixa al fitxer oddo.conf.

Anem pel següent, doncs!

Salut,

Jordi

El 7/3/23 a les 12:03, Fher ha escrit:
>
> Bon dia a tothom,
>
> Perdoneu el retard, però l'sprint 1 (temps per a preparar l'entorn de 
> desenvolupament i fer proves amb Odoo) va acabar el dia 28 i el dia 1 
> va començar l'sprint 2. Com ja sabeu, hem definit els sprints perquè 
> tinguin una durada d'un mes i durant aquest segon sprint ens toca el 
> següent:
>
>  1. Detallar les funcionalitats dels dos components que volem
>     desenvolupar amb prioritat, detallant què farà i com ho farà:
>     1.1 Control d'assistència i fitxa d'alumne: #2
>     <https://github.com/ElPuig/IMS/discussions/2>
>     1.2 Gestió d'actes: #3 <https://github.com/ElPuig/IMS/discussions/3>
>  2. Detallar els models de dades que necessitem: #1
>     <https://github.com/ElPuig/IMS/discussions/1>
>
> Aquests temes haurien de quedar tancats per poder començar l'sprint 3 
> el dia 1 d'abril.
>
> Salut i gràcies a tots,
> Fer
>
> —
> Reply to this email directly, view it on GitHub 
> <https://github.com/ElPuig/IMS/discussions/4>, or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AAUAHLZBKNQALUWRVEEQATLW24IWTANCNFSM6AAAAAAVSKF5DI>.
> You are receiving this because you are subscribed to this 
> thread.Message ID: ***@***.***>
>
-- 
Jordi Hernàndez
Tutor de 2n de DAM - INS Puig Castellar

**Comment by ChusVazquez** [comment]
Jo he començat a muntar, tal qual al Odoo, el model que vas proposar per la part d' assistencia, per veure si se m'acut algún camp extra o relació que no hem tingut en compte.

De moment, suposo que les UFs dependràn dels MP, que dependràn del estudis, etc...oi?

**Reply by FherStk** [reply]
La meva proposta:
Nivell (ESO, BTX, CCFF) -> Estudis (ESO, BTX, DAM, ASIX...) ->  MP (assumim que és una assignatura per ESO/BTX) -> UF (assumim que hi ha mínim una, per a ESO/BTX).

D'aquesta manera:
Els grups (ESO2, DAM2B, etc.) estan assignats a uns estudis (d'aquí es pot extreure el nivell).
Els alumnes estan assignats a UFs (d'aquí es pot extreure MP)

Els profes es poden assignar a UFs (docència) i a grups (tutors).

Què us sembla?

Salut!
Fer

**Comment by FherStk** [comment]
Perdoneu, però he tingut (i tinc) un munt de feina a sobre i no li he pogut dedicar gaire temps a l'IMS.

La setmana vinent és l'última de l'sprint, i potser és més fàcil per a tots que els que hàgiu pogut fer algun prototip ens el pogueu compartir. Tinc pensat el següent:

1. Penjar com a codi del repo l'esquelet d'un mòdul i un petit script que l'instal·li fent servir comandes del terminal (ja tinc alguna coseta feta dels experiments que vaig fer en cursos anteriors).
2. Cadascun de nosaltres hauria de fer una branca nova a partir d'aquest codi (recomano que les branques tinguin el vostre nom).
3. Els que heu tingut temps de fer un prototip, fora bo que l'adaptéssiu al codi de la vostra branca.
4. Un cop tinguem els prototips penjats a les branques, la resta els podrem estudiar i estudiar quins models de dades podem compartir. 

A veure si ho puc deixar tot llest, us dic quelcom.

Salut!
Fer
